corehq/apps/domain/forms.py

```diff
@@ -1322,17 +1323,14 @@ def clean_email(self):
             return self.cleaned_data['email']
 
 
-class HQSetPasswordForm(SetPasswordForm):
+class HQSetPasswordForm(EncodedPasswordChangeForm, SetPasswordForm):
```

Comment:
instead of changing this class would it be possible to create a new class and then only use the new class if enabled?
```
class EncodedPasswordComplianceForm(EncodedPasswordChangeFormMixin, HQSetPasswordForm):
    pass

form = EncodedPasswordComplianceForm if settings.ENABLE_PASSWORD_HASHING else HQSetPasswordForm
```